cvat/apps/engine/cache.py

```diff
@@ -275,6 +276,15 @@ def _delete_cache_item(self, key: str):
         except pickle.UnpicklingError:
             slogger.glob.error(f"Failed to remove item from the cache: key {key}", exc_info=True)
 
+    def _bulk_delete_cache_items(self, keys: Sequence[str]):
+        try:
+            self._cache().delete_many(keys)
+            slogger.glob.info(f"Removed chunks from the cache: keys {format_list(keys)}")
+        except pickle.UnpicklingError:
```

Comment:
I don't think that `pickle.UnpicklingError` can be raised here

Reply:
Indeed, removed handling for this error.